app/src/main/java/de/westnordost/streetcomplete/data/osm/edits/split_way/SplitWayAction.kt

```diff
@@ -131,9 +131,10 @@ private fun getSplitWayAtIndices(
 
     return nodesChunks.mapIndexed { index, nodes ->
         if (index == indexOfChunkToKeep) {
-            Way(originalWay.id, nodes, tags, originalWay.version, currentTimeMillis())
+            // keep the original timestampEdited, so resurvey quests are still shown after splitting
```

Comment:
as it refers to both Way creations, maybe keep it one level higher - before `if (index == indexOfChunkToKeep)`?

Reply:
The comment? makes sense